library/errata_tool_product.py

```diff
@@ -356,6 +381,7 @@ def run_module():
         state_machine_rule_set=dict(required=True),
         move_bugs_on_qe=dict(type='bool', default=False),
         text_only_advisories_require_dists=dict(type='bool', default=True),
+        exd_org_group=dict(choices=EXD_ORG_GROUPS.keys(), default='RHEL'),
```

Comment:
I ran this through some additional integration tests, and I found another issue here (apologies for catching this so late). We should remove `default=RHEL` in the argspec here, because if a playbook author does *not* set this `exd_org_group` parameter, then Ansible will automatically change an existing product's the `exd_org_group` value to "RHEL". It's ok to keep the documentation that says `default: RHEL` above, because that is how the ET server behaves when we don't POST any `exd_org_group_id`.

In fact, I want to remove most of these "default" values because they have a similar surprising behavior, https://github.com/ktdreyer/errata-tool-ansible/issues/129